legal-api/tests/unit/services/filings/validations/test_special_resolution.py

```diff
@@ -52,7 +54,7 @@
          '2010-01-10', HTTPStatus.BAD_REQUEST, 'Signatory given name is required.'),
         ('MISSING - signatory last name', 'some resolution', 'CP1234567', '2021-01-10', '2021-01-10', 'jane', None,
          '2010-01-10', HTTPStatus.BAD_REQUEST, 'Signatory family name is required.'),
-    ]
+    ], ids=lambda param: repr(param)[:75].replace('\'', '')
```

Comment:
If I didn't have this in here, github actions would be stuck printing out a million letters and cause the unit tests to run for 2-3H+